File: tests/Fortran/Free/0006.f90
program test
integer, parameter :: N = 10
real v1(N), v2(N), v3(N)

!$acc update host(v1)

!$acc update host(v1) &
!$acc& host(v2,v3)

!$acc update host(v1) &
!$acc host(v2) &
!$acc host(v3)

end program

File: tests/C/0006.c
int main (int argc, char *argv[])
{
	int v1[1000], v2[1000], v3[1000];

	#pragma acc update host(v1)

	#pragma acc update host(v1) \
	 host(v2,v3)

	#pragma acc update host(v1) \
	 host(v2) \
	 host(v3)

	return 0;
}
